src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+## 1.5.0-rc.1
```

Comment:
@reyang @cijothomas We are bumping the package versions of non-core packages to keep their major and minor versions in sync.

Reply:
Do we want rc or beta?

Reply:
I think we could use beta. The only concern with using beta is that users might feel a bit weird/uncomfortable when upgrading from an `rc` to a `beta` version (even though there is bump in the minor version).

Reply:
Which one (rc or beta) communicates the actual intention?

Reply:
I'm good with beta... it's been a bit of a lie that we've been releasing `rc`s this whole time as we've shipped a number of breaking changes in the past. Though I agree, that some folks may wonder why we've effectively downgraded from `rc` to `beta`.

Reply:
I think that would be beta. The reason I chose rc was that for the last 2.5 years, we have been releasing rc and suddenly moving people to beta might come across as retrograding.

I don't have a strong opinion here. I could change this to beta if folks want.

Reply:
Just a suggestion in the event we decide to go with `beta` https://github.com/open-telemetry/opentelemetry-dotnet/pull/4545#discussion_r1218705425

Reply:
I don't have strong opinions here, totally okay with whichever decision the maintainers agreed - 1) keep telling the lie if that's what most users want 2) stop telling the lie if the negative impact is reasonably small.

Reply:
Updated the version to `1.5.0-beta.1`